docs/releases/1.10.txt

```diff
@@ -138,7 +147,10 @@ File Uploads
 Forms
 ^^^^^
 
-* ...
+* Widget media gets now served from using ``staticfiles`` if
+  configured.
+* :meth:`django.forms.widgets.Media.absolute_path` removed
```

Comment:
This would be under "Backwards incompatible changes" not "Minor Features" but there's no need to mention private API changes unless you find evidence they are widely used.